docs/content/docs/recipes/react.md

```diff
@@ -5,247 +5,211 @@ aliases:
+In this tutorial, you'll learn about using the Fluid Framework by building a simple application that enables every client of the application to change a dynamic time stamp on itself and all other connected clients. You'll also learn how to connect the Fluid data layer with a view layer made in [React](https://reactjs.org/).
 
-![Four browsers with the Timestamp app open in them.](https://fluidframework.blob.core.windows.net/static/images/Four-clients-1.PNG)
+The following image shows finished application loaded into four browsers. Each browser contains a button and a timestamp. When the button is pressed in any of the browsers, the timestamp is updated in all four browsers.
 
-The following image shows the same four clients one second after the **click** button was pressed. Note that the timestamp has updated to the very same time in all four browsers.
-
-![Four browsers with the Timestamp app open in them one second after the button has been pushed.](https://fluidframework.blob.core.windows.net/static/images/Four-clients-2.PNG)
+![cra](https://user-images.githubusercontent.com/1434956/111496992-faf2dc00-86fd-11eb-815d-5cc539d8f3c8.gif)
 
 {{< callout note >}}
 
 This tutorial assumes that you are familiar with the [Fluid Framework Overview]({{< relref "/docs/_index.md" >}}) and that you have completed the [QuickStart]({{< relref "quick-start.md" >}}). You should also be familiar with the basics of [React](https://reactjs.org/), [creating React projects](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app), and [React Hooks](https://reactjs.org/docs/hooks-intro.html).
 
 {{< /callout >}}
 
-## Create the project
+Concepts you will learn:
+1. How to integrate Fluid into a React application
+2. How to run and connect your application to a local Fluid service (Tinylicious)
+3. How to create and get Fluid Containers and collaborative objects
+4. How to use a [SharedMap distributed data structure (DDS)](https://fluidframework.com/docs/apis/map/sharedmap/) to sync data between connected clients
 
-1. Open a Command Prompt and navigate to the parent folder where you want to create the project; e.g., `c:\My Fluid Projects`.
-1. Run the following command at the prompt. (Note that the CLI is np**x**, not npm. It was installed when you installed Node.js.)
 
-    ```dotnetcli
-    npx create-react-app fluid-react-tutorial --use-npm
-    ```
+\* Just want to see the code? Jump to the [finished tutorial.](https://github.com/microsoft/FluidExamples/blob/main/cra-demo/src/App.js).
 
-1. The project is created in a subfolder named `fluid-react-tutorial`. Navigate to it with the command `cd fluid-react-tutorial`.
-1. The project uses two Fluid libraries:
```

Comment:
Why remove this?

Reply:
it was older content. the new demo has the NPX command lower down.

Reply:
i was able to return much of this content as it doesn't conflict with the new approach